spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DockerComposePostgresRabbitInfluxIT.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.integration.test;
+
+import com.palantir.docker.compose.DockerComposeRule;
+import com.palantir.docker.compose.configuration.DockerComposeFiles;
+import com.palantir.docker.compose.connection.DockerMachine;
+import com.palantir.docker.compose.connection.waiting.HealthChecks;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.cloud.dataflow.integration.test.util.ClosableDockerComposeRule;
+
+/**
+ * IT Tests with Postgres, InfluxDB and RabbitMQ
+ *
+ * @author Christian Tzolov
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+		DockerComposeTestPlatform.class,
+		DockerComposeTestStream.class,
+		DockerComposeTestTask.class
+})
+public class DockerComposePostgresRabbitInfluxIT {
+
+	private static final Logger logger = LoggerFactory.getLogger(DockerComposePostgresRabbitInfluxIT.class);
+
+	@ClassRule
```

Comment:
We should try to use juni5 for these and see if ported rule in spring-cloud-dataflow-common-test-docker works.